sklearn/preprocessing/data.py

```diff
@@ -34,8 +34,9 @@
 from ..utils.validation import (check_is_fitted, check_random_state,
                                 FLOAT_DTYPES)
 
-from ._encoders import OneHotEncoder
+from ._csr_expansion import _csr_expansion
```

Comment:
Please rename both the module and the function to `_csr_polynomial_expansion` to make it more explicit.